project/AppDependencies.scala

```diff
@@ -16,7 +16,7 @@ object AppDependencies {
     "uk.gov.hmrc"       %% "play-frontend-hmrc"         % "2.0.0-play-28",
     "uk.gov.hmrc"       %% "mongo-caching"              % "7.0.0-play-28",
     "uk.gov.hmrc"       %% "play-language"              % "4.13.0-play-28",
-    "org.typelevel"     %% "cats-core"                  % "2.3.1",
+    "org.typelevel"     %% "cats-core"                  % "2.7.0",
```

Comment:
any special reason for the upgrade?

Reply:
no, just checking a single library upgrade doesn't break anything on the basis that forced multiple library upgrades can be time consuming